### ISSUE TITLE
Improve kiwix-build README

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-build#3]
README should clearly explain the compilation procedure. This means step-by-step procedures which can be follow blindly with short explanation about what is going on:
* What are the dependencies (with minimal version)
* How to compile kiwix-lib and kiwix-tools under Linux static/dynamic by downloading locally deps
* How to cross-compile kiwix-lib and kiwix-tools under Linux for Windows static/dynamic by downloading locally deps

**Comment by mgautierfr** [comment]
I've merged the branch `android` into `master` and with that, I've rewrite the README. Can you check it on your side (and push changes in a new PR if necessary) ?

**Comment by kelson42** [comment]
@mgautierfr thx, will have a look.

**Comment by kelson42** [comment]
@mgautierfr I have started to rewrite the README and I have a few questions:
* How to get the list of target? How to compile a Kiwix APK for Android?
* How to compile for linux arm?

**Comment by ynikitenko** [comment]
Please write that `qmake` must be installed. I had an error in the middle of building that qmake couldn't be found. I think that could be better reported in the very beginning (or, in a worse case, in README).
Also ninja could be automatically installed from pip. Why don't we add that to setup.py requirements? I think that should work fine in most cases.

**Comment by ynikitenko** [comment]
This was suggested earlier, but the issue was closed (https://github.com/kiwix/kiwix-build/issues/49). I think there are two options: either do that automatically, or include in README (as proposed in this issue).

**Comment by kelson42** [comment]
> Please write that `qmake` must be installed. I had an error in the middle of building that qmake couldn't be found. I think that could be better reported in the very beginning (or, in a worse case, in README).
> Also ninja could be automatically installed from pip. Why don't we add that to setup.py requirements? I think that should work fine in most cases.

@ynikitenko `qmake` is de dependence only used for Kiwix Desktop. Qt is a clear dependence (written in README.md) of Kiwix Desktop already. We don't repeat here what is written in specific project READMEs.

**Comment by mgautierfr** [comment]
> So it seems that kiwix-build promises to deal with all dependencies itself? It would be great.

Yes. And that is THE purpose of kiwix-build. But we are speaking about run-time dependency (either by compiling them or using the system packages)
For build dependencies (qmake, cmake, ninja,..) we use the system packages.

There is already a system to install needed packages (deb, rpm). If some needed packages are not installed by kiwix-build, please report it in new issues.

**Comment by ynikitenko** [comment]
@mgautierfr now I understand, thanks. But in the prerequisites section it's written 

> You will need a recent version of Meson (>= 0.34) and Ninja (>= 1.6)

Why don't you add `qmake` there?
